src/screens/Sablier/Withdraw.js

```diff
@@ -78,31 +75,41 @@ const FooterWrapper = styled.View`
   width: 100%;
 `;
 
+const InputWrapper = styled.View`
+  padding: 16px 40px 0;
+`;
+
 const Withdraw = (props: Props) => {
   const {
     calculateSablierWithdrawTransactionEstimate,
     navigation,
     assets,
     supportedAssets,
-    baseFiatCurrency,
-    rates,
     withdrawTransactionEstimate,
     isCalculatingWithdrawTransactionEstimate,
     useGasToken,
     balances,
   } = props;
 
-  const [withdrawAmountInWei, setWithdrawAmount] = useState(0);
+  const [withdrawAmount, setWithdrawAmount] = useState('');
```

Comment:
Per earlier comment – can we keep it as `0`?

Reply:
Not really - if I add `'0'` here (string as it's a text field everywhere) then it would add leading zero. So if user inputs "1" he'll get "01". Not cool.
The initial state `''` means that user just hasn't typed anything yet = the text input is empty so it's a valid use case imho. And empty value will be detected in validation anyway. Setting it to null or 0 will add unnecessary complexity.